cocos/audio/openharmony/PcmAudioService.cpp

```diff
@@ -47,7 +47,7 @@ PcmAudioService::~PcmAudioService() {
 }
 
 
-int32_t PcmAudioService::AudioRendererOnWriteData(OH_AudioRenderer* renderer,
+OH_AudioData_Callback_Result PcmAudioService::AudioRendererOnWriteData(OH_AudioRenderer* renderer,
```

Comment:
I think we can change it to a global function later, so there's no need to reference it in the header file